server/index.ts

```diff
@@ -144,6 +144,12 @@ export const configSchema = schema.object({
       buttonstyle: schema.string({ defaultValue: '' }),
     }),
   }),
+  kerberos: schema.object({
+    jwt_siging_key: schema.string({
+      defaultValue: 'secret share between opensearch and dashboards',
```

Comment:
can you change the default value to be empty?

Reply:
+1 
This is a placeholder, but defaults like this should never be hardcoded in production